app/src/main/java/agency/five/codebase/android/movieapp/navigation/NavigationItem.kt

```diff
@@ -0,0 +1,27 @@
+package agency.five.codebase.android.movieapp.navigation
+
+import agency.five.codebase.android.movieapp.R
+
+const val HOME_ROUTE = "Home"
+const val FAVORITES_ROUTE = "Favorites"
+
+sealed class NavigationItem(
+    override val route: String,
+    val selectedIconId: Int,
+    val unselectedIconId: Int,
+    val labelId: Int,
+) : MovieAppDestination(route) {
+    object HomeDestination : NavigationItem(
+        route = HOME_ROUTE,
+        selectedIconId = R.drawable.homeicon_selected,
```

Comment:
homeicon? those are two words, and before every new word `_` should go

please check all resources